service.go

```diff
@@ -133,6 +133,9 @@ type Config struct {
 	//    - ReloadSignal  string () [USR1, ...]     - Signal to send on reaload.
 	//    - PIDFile       string () [/run/prog.pid] - Location of the PID file.
 	//    - LogOutput     bool   (false)            - Redirect StdErr & StdOut to files.
+	//  * Windows
+	//    - StartType     int (2) - win32 CreateService dwStartType option, 2 = SERVICE_AUTO_START
```

Comment:
Docs at: https://docs.microsoft.com/en-us/windows/desktop/api/winsvc/nf-winsvc-createservicew

All the start types:

SERVICE_AUTO_START = 0x00000002
SERVICE_BOOT_START = 0x00000000
SERVICE_DEMAND_START = 0x00000003
SERVICE_DISABLED = 0x00000004
SERVICE_SYSTEM_START = 0x00000001

System_Start and Boot_Start don't make sense for this package. I'm not sure why someone would install a disabled service. OnDemand might be done. Could we just make this a bool `OnDemand (bool) (false)`? otherwise assume Auto_Start.